python/TestHarness/TestHarness.py

```diff
@@ -1019,7 +1019,7 @@ def parseCLArgs(self, argv):
         parser.add_argument('--error', action='store_true', help='Run the tests with warnings as errors (Pass "--error" to executable)')
         parser.add_argument('--error-unused', action='store_true', help='Run the tests with errors on unused parameters (Pass "--error-unused" to executable)')
         parser.add_argument('--error-deprecated', action='store_true', help='Run the tests with errors on deprecations')
-
+        parser.add_argument('--warn-unused',action='store_true', help='Run the tests without errors on unused parameters (Pass "--warn-unused" to executable)')
```

Comment:
Just making sure we need this: Without diving in, the case would be to downgrade an application that has chosen to default to "error_unused" right?

If that's the case, let's make sure we test _this_ case.